src/components/ui/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,8 +1,19 @@
-import { Breadcrumb as UIBreadcrumb } from '@faststore/ui'
 import React, { memo } from 'react'
-import Icon from 'src/components/ui/Icon'
+import {
+  Breadcrumb as UIBreadcrumb,
+  Dropdown as UIDropdown,
+  DropdownButton as UIDropdownButton,
+  DropdownItem as UIDropdownItem,
+  DropdownMenu as UIDropdownMenu,
+} from '@faststore/ui'
 import Link from 'src/components/ui/Link'
 import type { BreadcrumbProps as UIBreadcrumbProps } from '@faststore/ui'
+import Icon from 'src/components/ui/Icon'
+import { navigate } from 'gatsby'
+
+import './breadcrumb.scss'
```

Comment:
After some recent changes, we're now importing the component's CSS in the file `components.scss`.